### PR TITLE
[Title]2nd Documentation

### PR DESCRIPTION
**Pull request by shohagHub**
Signed-off-by: Saleh Enam Shohag <shohag402@gmail.com>

### CODE REVIEW

File: Documentation/Documentation/HealthDataStore.swift [hunk 1]
```diff
@@ -24,7 +25,23 @@ public class HealthDataStore{
...
+            let alert = UIAlertController(title: "APPLICATION_NAME", message: message , preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Ok", style: UIAlertAction.Style.default, handler: okClickHandler))
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.cancel, handler: nil))
+            VC.present(alert, animated: true, completion: nil)
+        }
+    }
+    
+    /*
+     Call this function to add two numbers
+     */
```
Comment (shohaggithub2) [inline Documentation/Documentation/HealthDataStore.swift:40]: write the comment accurately.

File: Documentation/Documentation/ViewController.swift [hunk 2]
```diff
@@ -17,4 +17,3 @@ class ViewController: UIViewController {
 
 
 }
```
Comment (shohaggithub2) [inline Documentation/Documentation/ViewController.swift:19]: Why is this empty file change added.

File: Documentation/Documentation/HealthDataStore.swift (hunk 1 above)
Comment (shohagHub) [reply Documentation/Documentation/HealthDataStore.swift:40]: In next cl I will add the accurate comment.

File: Documentation/Documentation/ViewController.swift (hunk 2 above)
Comment (shohagHub) [reply Documentation/Documentation/ViewController.swift:19]: it is mistakenly added.

**Review by shohaggithub2 — APPROVED** [review]

**Pull request merged**